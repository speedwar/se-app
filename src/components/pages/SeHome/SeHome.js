import React from 'react'
import styled from 'styled-components'
import { SeLayout } from 'components/layouts'
import { SeCard } from 'components/modules'
import { enau } from 'components/i18n'

import imgTile from 'assets/images/tile.jpg'
import imgLogo from 'assets/images/logo.png'

const CardContainer = styled.div`
  width: 100%;
  height: 700px;
  display: flex;
  justify-content: center;
  align-items: center;
`

const Content = () => (
  <>
    <CardContainer>
      <SeCard
        imgLogo={ imgTile }
        imgTile={ imgLogo }
        title={ enau.programs.homeandaway.title }
      />
    </CardContainer>
  </>
)

const SeHome = () => (
  <SeLayout>
    <Content />
  </SeLayout>
)

export default SeHome
