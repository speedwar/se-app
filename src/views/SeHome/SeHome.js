import React from 'react'
import { SeLayout } from 'layouts'
import { SeProgram } from 'components'

const Content = () => (
  <>
    <SeProgram title="Programs" />
  </>
)

const SeHome = () => (
  <SeLayout>
    <Content />
  </SeLayout>
)

export default SeHome
