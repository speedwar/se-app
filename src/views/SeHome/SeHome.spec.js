import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'
import SeHome from './SeHome'

const setup = () => {
  const wrapper = shallow(<SeHome />)
  return { wrapper }
}

describe('<SeHome /> component', () => {
  it('renders component', () => {
    const wrapper = setup()
    expect(wrapper).toMatchSnapshot()
  })
})
