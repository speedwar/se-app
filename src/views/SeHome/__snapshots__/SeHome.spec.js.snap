// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SeHome /> component renders component 1`] = `
Object {
  "wrapper": <SeLayout>
    <Content />
  </SeLayout>,
}
`;
